compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -497,7 +497,12 @@ impl<'value, 'interner> Display for ValuePrinter<'value, 'interner> {
             Value::Quoted(tokens) => {
                 write!(f, "quote {{")?;
                 for token in tokens.iter() {
-                    write!(f, " {token}")?;
+                    match token {
+                        Token::QuotedType(id) => {
+                            write!(f, " {:?}", self.interner.get_quoted_type(*id))?;
```

Comment:
```suggestion
                            write!(f, " {}", self.interner.get_quoted_type(*id))?;
```
^ should this be `Display`ed?

Reply:
Yep